KubeArmor/monitor/logUpdate.go

```diff
@@ -142,9 +143,10 @@ func (mon *SystemMonitor) UpdateLogs() {
 					log = execLogMap[log.HostPID]
 					delete(execLogMap, log.HostPID)
 
-					if msg.ContextSys.Retval == -13 {
```

Comment:
Just checking what does the hardwired value of -13 does here.. Is there a better way to represent this with a constant?

Reply:
-13 is the error code defined in the Linux kernel headers for "Permission denied".